scipy/sparse/linalg/isolve/lsqr.py

```diff
@@ -298,14 +302,17 @@ def lsqr(A, b, damp=0.0, atol=1e-8, btol=1e-8, conlim=1e8,
 
     """
     Set up the first vectors u and v for the bidiagonalization.
-    These satisfy  beta*u = b,  alfa*v = A'u.
+    These satisfy  beta*u = b - A*x,  alfa*v = A'*u.
```

Comment:
Nit: asterisks have special meaning for Sphinx, need to be shielded by double backticks or (IMO better) making it a proper formula with a `:math: ` directive.